MAUI/DataForm/Overview.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Overview of .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the introduction of Syncfusion .NET MAUI DataForms(SfDataForm) control and its key features.
+platform: Maui
```

Comment:
platform: Maui => change platform: maui
due to CI issue

Reply:
Maui to maui changed